lib/src/filters.dart

```diff
@@ -2,5 +2,5 @@ import 'package:pub_semver/pub_semver.dart';
 
 final _dart2 = VersionConstraint.parse('>=2.0.0');
 
-bool isDart2(String sdkVersion) =>
-    VersionConstraint.parse(sdkVersion).allowsAny(_dart2);
+bool isDart2([String sdkVersion]) =>
+    VersionConstraint.parse(sdkVersion ?? 'any').allowsAny(_dart2);
```

Comment:
👍 

thanks!

Reply:
No problem! Not sure why this is getting a null value though, maybe some bogus metadata when downloading?